tests/sentry/issues/endpoints/test_project_stacktrace_link.py

```diff
@@ -374,74 +368,32 @@ def test_munge_flutter_worked(self, mock_integration: MagicMock) -> None:
 class ProjectStacktraceLinkTestMultipleMatches(BaseProjectStacktraceLink):
     def setUp(self) -> None:
         BaseProjectStacktraceLink.setUp(self)
-        # Created by the user, not well defined stack root
```

Comment:
Moving these into the specific test since I will write a new test that will not use all of these.